lib/spack/spack/cmd/edit.py

```diff
@@ -67,68 +119,72 @@ def edit_package(name, repo_path, namespace, force=False):
     spec = Spec(name)
     if os.path.exists(path):
         if not os.path.isfile(path):
-            tty.die("Something's wrong.  '%s' is not a file!" % path)
+            tty.die("Something is wrong. '{0}' is not a file!".format(path))
         if not os.access(path, os.R_OK | os.W_OK):
             tty.die("Insufficient permissions on '%s'!" % path)
     elif not force:
-        tty.die("No package '%s'.  Use spack create, or supply -f/--force "
-                "to edit a new file." % spec.name)
+        tty.die("No package for '{0}' was found.".format(spec.name),
+                "  Use `spack create` to create a new package")
     else:
+        # Force create a new package (deprecated)
         mkdirp(os.path.dirname(path))
-        with open(path, "w") as pkg_file:
-            pkg_file.write(
-                package_template.substitute(
-                    name=spec.name, class_name=mod_to_class(spec.name)))
+        with open(path, 'w') as pkg_file:
+            pkg_file.write(package_template.format(
+                name=spec.name, class_name=mod_to_class(spec.name)))
 
     spack.editor(path)
 
 
 def setup_parser(subparser):
     subparser.add_argument(
         '-f', '--force', dest='force', action='store_true',
-        help="Open a new file in $EDITOR even if package doesn't exist.")
+        help='deprecated, use `spack create` instead')
```

Comment:
Is this deprecated enough for now? Honestly, I would be fine with ripping it out completely in this PR, but I can wait.